lib/modules/apostrophe-schemas/views/macros.html

```diff
@@ -205,6 +205,7 @@
 {%- endmacro -%}
 
 {%- macro singletonBody(field) -%}
```

Comment:
Does this mean this only covers singletons and not areas generally?

Reply:
Seems like the macro `area` uses `singletonBody`, but I might be missing something

```nunjunks
{%- macro area(field) -%}
  {{ fieldset(field | merge({ attributes: 'data-editable' }), singletonBody) }}
{%- endmacro -%}
```